src/lib/suit.ts

```diff
@@ -1,12 +1,16 @@
 const NAMESPACE = 'ais';
 
-export const component = (componentName: string) => ({
-  descendantName,
-  modifierName,
-}: {
+interface SuitNames {
```

Comment:
This name is confusing because it implies that it's an array. Would `SuitOptions` make more sense here? (that's one drawback of naming things)

Reply:
No objection to rename it (I was not convinced by the name myself :neutral_face:  )